site/content/internal/sustained-engineering.md

```diff
@@ -83,7 +83,15 @@ During non-release weeks, triage is held twice a week. Near the end of the relea
 
 ## Support Responses
 
-Part of SET's responsibility is to interface with the customer support team. SET's primary goal in respect to support is to triage escalated support issues, work on any high priority customer bugs requiring code change, as well as provide training and knowledge share with the customer support team. To help us do this better, SET members should use the following responses when receiving issue escalations from the customer support team:
+Part of SET's responsibility is to interface with the customer support team. SET's primary goal in respect to support is to triage escalated support issues, work on any high priority customer bugs requiring code change, as well as provide training and knowledge share with the customer support team. When customer support team requires enginerring support, they should:
```

Comment:
```suggestion
Part of SET's responsibility is to interface with the customer support team. SET's primary goal in respect to support is to triage escalated support issues, work on any high priority customer bugs requiring code change, as well as provide training and knowledge share with the customer support team. When customer pre-sales support team looks for engineering support, they should:
```

Reply:
^I think Elias means both support and solution architects can use this process as above

Reply:
Suggestion for final sentence: When the pre-sales customer support team requires engineering support, they should:

Reply:
Thanks! Updated per suggestion